base/src/androidLibMain/kotlin/utils/Locale.kt

```diff
@@ -39,6 +39,8 @@ actual class Locale internal constructor(val locale: java.util.Locale) {
         get() = locale.script
     actual val variantCode: String
         get() = locale.variant
+    actual val usesMetricSystem: Boolean
+        get() = !listOf("US", "GB", "MM", "LR").contains(countryCode.toUpperCase(java.util.Locale.ROOT))
```

Comment:
Doesn't the United Kingdom use the metric system?

Reply:
Well, they're using a mix.
Maybe it better to use this, but we still have this fallback for early API versions:
https://developer.android.com/reference/kotlin/android/icu/util/LocaleData.MeasurementSystem